app/models/form/common/questions/created_by_id.rb

```diff
@@ -11,8 +11,8 @@ def answer_options
     answer_opts = { "" => "Select an option" }
     return answer_opts unless ActiveRecord::Base.connected?
 
-    User.select(:id, :name).each_with_object(answer_opts) do |user, hsh|
-      hsh[user.id] = user.name
+    User.select(:id, :name, :email).each_with_object(answer_opts) do |user, hsh|
```

Comment:
pretty sure you will want different scope here depending on the user viewing this question otherwise the data coordinator will see everyone

Reply:
I'm pretty sure this should be restricted to users within their org, or perhaps child/parent orgs.